flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/DataGenerators.java

```diff
@@ -70,6 +78,12 @@ public static class Primitives implements DataGenerator {
     private static final BigDecimal BIG_DECIMAL_NEGATIVE = new BigDecimal("-1.50");
     private static final byte[] FIXED_BYTES = "012345689012345".getBytes(StandardCharsets.UTF_8);
 
+    private static org.apache.avro.Schema addAdjustToUtc(
```

Comment:
Do we need it? It doesn't seem to have been used.

Reply:
good catch. this is not needed anymore.